cs.json

```diff
@@ -0,0 +1,348 @@
+{
+    "name" : "Čeština",
+    "name-en": "Czech",
+    "live" : true,
+    "translations" : {
+        "meta" : {
+            "thForDate" : false,
+            "dateFormat" : "[date]. [month] [year]",
+            "months" : [
+                "Leden",
+                "Únor",
+                "Březen",
+                "Duben",
+                "Květen",
+                "Červen",
+                "Červenec",
+                "Srpen",
+                "Září",
+                "Říjen",
+                "Listopad",
+                "Prosinec"
+            ],
+            "types" : {
+                "backpack" : "Batoh",
+                "emote" : "Akce",
+                "glider" : "Padák",
+                "emoji" : "Emotikon",
+                "loading" : "Načítací obrazovka",
+                "outfit" : "Oděv",
+                "pickaxe" : "Krumpáč",
+                "skydive" : "Stopa pádu",
+                "umbrella" : "Deštník",
+                "spray" : "Sprej",
+                "toy" : "Hračka",
+                "pet" : "Mazlíček",
+                "music" : "Hudba",
+                "wrap" : "Plášť",
+                "banner" : "Banner", //Could be banner or plakát, depending on where it is used
```

Comment:
Unfortunately, Fortnite is unavailable in the Czech language, so we can't follow official translations. I've done some research myself and I believe "Banner" is appropriate in this context.